easycukes-rest/src/main/java/com/worldline/easycukes/rest/stepdefs/RESTStepdefs.java

```diff
@@ -448,4 +447,24 @@ public void checkIfParameterEquals(String parameter, String value)
     public void setDateFormat(String format) throws Throwable {
         CukesHelper.setParameter(RestConstants.DATE_FORMAT, format);
     }
+    
+    @Given("^I set the parameter \"(.*?)\" to first not null property \"(.*?)\" in response array$")
+	public void setAParameterToAFirstNotNullPropertyFromResponseArray(final String param, final String property)
+			throws ParseException {
+		CukesHelper.setParameter(param, RestService.getInstance().getIdFromResponseArrayByProperty(property));
+	}
+
+
+	@When("^I send put to \"(.*?)\" the parameters:$")
```

Comment:
Could we have something like `I send a put request to ... with parameters ..."?

Just for it to be more readable?